src/components/mainContent/Navbar.css

```diff
@@ -0,0 +1,40 @@
+.dropdown {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 450px;
+}
+
+.dropdown-content {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0 0 10px 10px;
+  display: none;
+  position: absolute;
+  background-color: #f7f7f7;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px 898F9C;
```

Comment:
Should the color here be #898F9C instead?

Reply:
The color is #f7f7f7 there because the background of the dropdown pre-hover is a similar shade to the body!

Reply:
Sorry, I meant for the `box-shadow` line - there's no `#` in front of the color

Reply:
Ah, I see, I must've forgotten to include it. It'll be there in the next commit!